README.md

```diff
@@ -93,51 +87,77 @@ Distributed under the AGPLv3 License. Read more [here](https://www.getlago.com/b
 | **Lago Ruby Client**     | [![Lago Ruby Client Release](https://img.shields.io/github/v/release/getlago/lago-ruby-client)](https://github.com/getlago/lago-ruby-client/releases) |
 
 
-## 💻 Deploy locally
+# Lago
+
+Lago is an open-source metering and usage-based billing API designed to help businesses manage consumption tracking, subscription management, pricing iterations, payment orchestration, and revenue analytics. It serves as a robust alternative to platforms like Chargebee, Recurly, or Stripe Billing, catering to various pricing models from usage-based to subscription-based structures.
+
+## 💻 Deploy Locally
 
 ### Requirements
-1. Install Docker on your machine;
-2. Make sure Docker Compose is installed and available (it should be the case if you have chosen to install Docker via Docker Desktop); and
-3. Make sure Git is installed on your machine.
+To deploy Lago locally, ensure that the following dependencies are installed on your machine:
+
+1. **Docker** - Install [Docker](https://docs.docker.com/get-docker/).
+2. **Docker Compose** - Docker Desktop includes Docker Compose by default, but verify its availability.
+3. **Git** - Install [Git](https://git-scm.com/downloads) if it is not already installed.
 
-### Run the app
-To start using Lago, run the following commands in a shell:
+### Run the App
+To start using Lago, execute the following commands in your terminal:
 
+#### 1️⃣ Clone the Repository
 
-#### On a fresh install
 ```bash
-# Get the code
 git clone --depth 1 https://github.com/getlago/lago.git
-
-# Go to Lago folder
 cd lago
+```
+This will download the Lago project and navigate into its directory.
+
+#### 2️⃣ Configure Environment Variables
 
-# Set up environment configuration
+```bash
 echo "LAGO_RSA_PRIVATE_KEY=\"`openssl genrsa 2048 | base64`\"" >> .env
 source .env
+```
+This generates a new RSA private key and stores it in the `.env` file. It then loads the environment variables into your shell session.
+
+#### 3️⃣ Start the API Service
 
-# Start the api
+```bash
 docker compose up -d api
+```
+This command runs the API service in detached mode (`-d`), meaning it will run in the background without blocking your terminal.
 
-# Create the database
+#### 4️⃣ Set Up the Database
+
+```bash
 docker compose exec api rails db:create
```

Comment:
Also we can add a note that those are already done when we run `docker compose up -d api`